releasenotes/releasenotes_v5.rst

```diff
@@ -0,0 +1,164 @@
+.. _vonk_releasenotes_history_v5:
+
+Current Firely Server release notes (v5.x)
+==========================================
+
+.. _vonk_releasenotes_5_0_0:
+
+Release 5.0.0-beta1, TBD, 2023
+------------------------------
+.. attention::
+    This is a beta release of Firely Server 5.0.0. Although the core functionality remains fully intact, parts of the public API have been removed or heavily modified. Please consult the list under section 'Plugin and Facade' and check whether your implementation is affected by these changes.
+
+Feature
+^^^^^^^
+
+#. The default information model for Firely Server is now R4.
+#. Firely Server is now certified according to the `ONC Certification (g)(10) Standardized API <https://inferno.healthit.gov/suites/test_sessions/10cad0a0-1d6e-4648-b2c8-70cefbf260c5>`_.
+#. Bulk Data Export now supports SMART on FHIR v2.
+#. Contents of AuditEvents can now be modified via a plugin. See :ref:`AuditEvent customization <audit_event_customization>` for further info.
+
+Fix
+^^^
+
+#. Bulk Data Export now returns a succesful status code (``202``) instead of an erroneous status code if no resources were matched for an export. The resulting export will include an empty array as described in the `specification <https://hl7.org/fhir/uv/bulkdata/export/index.html#response---complete-status>`_.
+#. Empty search parameters are now ignored by the server instead of resulting in an error response.
+#. Firely Server now creates valid R5 AuditEvents.
+#. Firely Server now supports searching on version-specific references. Consult the `FHIR specification <https://www.hl7.org/fhir/search.html#versions>`_ for more information.
+#. Searching for a resource with multiple sort fields does not throw an exception anymore.
+#. Fields that are included in the audit event log and AuditEvent resources now contain the same content.
```

Comment:
I would move this to features: "Firely Server now generated FHIR AuditEvent resources conforming to IHE Basic Audit Log Patterns. See https://profiles.ihe.net/ITI/BALP/index.html". Fields that are included in the audit event log and AuditEvent resources now contain the same content.

Reply:
Done